.github/docker/centreon-web/alma9/Dockerfile.dependencies

```
@@ -106,6 +106,20 @@ dnf install -y \
   php-pear \
   php-json
 
+###################
+# install plugins #
+###################
+
+dnf install -y centreon-plugin-Operatingsystems-Linux-Snmp nrpe centreon-plugin-Operatingsystems-Linux-Local.noarch nagios-plugins-nrpe
+mkdir -p /var/lib/centreon/centplugins/ \
+    && chown nrpe: /var/lib/centreon/centplugins/ \
+    && sed -i 's/dont_blame_nrpe=0/dont_blame_nrpe=1/' /etc/nagios/nrpe.cfg \
+    && echo 'command[check_centreon_plugins]=/usr/lib/centreon/plugins/centreon_linux_local.pl --plugin=\$ARG1\$ --mode=\$ARG2\$ \$ARG3\$' > /etc/nrpe.d/centreon-commands.cfg \
+    && systemctl restart nrpe \
+    && systemctl enable nrpe \
+    && usermod -a -G systemd-journal nrpe \
+    && systemctl restart nrpe
```

IMHO we should only add linux snmp plugin because snmpd is already installed in the container and should only be started
```suggestion
dnf install -y centreon-plugin-Operatingsystems-Linux-Snmp
```

---

The code you removed install the nrpe server, linux snmp plugin and  linux nrpe plugin

---

Yep but the nrpe stuff is to monitor the disk service. SNMP is not able to monitor the disk on Docker (OID not available)

---

Will try the storage mode

---

Ok the storage mode works but it does not monitor the /. Here is the output
```
OK: All storages are ok | 'count'=10;;;0; 'used_/dev'=0B;;;0;67108864 'used_/dev/shm'=0B;;;0;67108864 'used_/etc/resolv.conf'=15538511872B;;;0;1081100128256 'used_/etc/hosts'=15538511872B;;;0;1081100128256 'used_/etc/hostname'=15538511872B;;;0;1081100128256 'used_/proc/kcore'=0B;;;0;67108864 'used_/proc/keys'=0B;;;0;67108864 'used_/proc/timer_list'=0B;;;0;67108864 'used_/proc/scsi'=0B;;;0;4108787712 'used_/sys/firmware'=0B;;;0;4108787712
```

---

If you really need to monitor `/` partition, then you can use `centreon-plugin-Operatingsystems-Linux-Local` without using nrpe